### DISCUSSION TITLE
Authentication Token Error

### DISCUSSION THREAD

**Discussion by samecfrank** [pyannote/pyannote-audio#1190] [General]
I am using pyannote.audio for speaker diarization by running the code shown below. 
```
pipeline = pyannote.audio.Pipeline.from_pretrained("pyannote/speaker-diarization@2.1",use_auth_token = auth_token)
diarization = pipeline(file_name,num_speakers = num_speakers)
```
I followed the steps given on the hugging face website for making an authentication token (see below)
```
# 1. visit hf.co/pyannote/speaker-diarization and accept user conditions
# 2. visit hf.co/pyannote/segmentation and accept user conditions
# 3. visit hf.co/settings/tokens to create an access token
# 4. instantiate pretrained speaker diarization pipeline
```
However, I am still getting the following error. Does anyone know why this may be? Thank you! 

`RepositoryNotFoundError: 401 Client Error: Repository Not Found for url: https://huggingface.co/pyannote/segmentation/resolve/2022.07/pytorch_model.bin. If the repo is private, make sure you are authenticated. (Request ID: Root=1-639750ac-45de6da3313b2ee6787c69cb)`

**Comment by NathanielParizi** [comment]
I'm experiencing similar issue. I understand that you need to be authenticated by accepting the user conditions. 

I can't seem to find anything to accept them at: hf.co/pyannote/speaker-diarization
I have accepted at: hf.co/pyannote/segmentation

I still get the stacktrace: 

`Traceback (most recent call last):
  File "path/whisper/main.py", line 40, in <module>
    diarization = pipeline(file_path)
TypeError: 'NoneType' object is not callable

Could not download 'pyannote/speaker-diarization' pipeline.
It might be because the pipeline is private or gated so make
sure to authenticate. Visit https://hf.co/settings/tokens to
create your access token and retry with:

   >>> Pipeline.from_pretrained('pyannote/speaker-diarization',
   ...                          use_auth_token=YOUR_AUTH_TOKEN)

If this still does not work, it might be because the pipeline is gated:
visit https://hf.co/pyannote/speaker-diarization to accept the user conditions.

Process finished with exit code 1`

This thread seems to experienced a similar issue:
https://github.com/openai/whisper/discussions/264

**Reply by NathanielParizi** [reply]
Good news guys. I resolved the issue on my end. I simply didn't log into my huggingface account. 
Please give it a try: 

Solution)

Update the PyAnnote version to ‘2.1.1’ using ‘pip install --upgrade pyannote.audio’
visit “[pyannote/speaker-diarization · Hugging Face 18](https://huggingface.co/pyannote/speaker-diarization)” and accept user conditions
visit “[pyannote/segmentation · Hugging Face 13](https://huggingface.co/pyannote/segmentation)” and accept user conditions
Generate, only ‘READ’ token using your Huggingface account settings–>access Tokens
Login to Huggingface through the command line using the command “huggingface-cli login”. This will ask you the token created in step 4 and then append this token under your ‘.huggingface’ folder.
